tests/cli/test_rasa_data.py

```diff
@@ -340,6 +340,96 @@ def test_rasa_data_convert_nlg_to_yaml(
     )
 
 
+def test_rasa_data_convert_responses(
+    run_in_simple_project: Callable[..., RunResult], run: Callable[..., RunResult]
+):
+    converted_data_folder = "converted_data"
+    os.mkdir(converted_data_folder)
+    expected_data_folder = "expected_data"
+    os.mkdir(expected_data_folder)
+
+    domain = (
+        "version: '2.0'\n"
+        "session_config:\n"
+        "  session_expiration_time: 60\n"
+        "  carry_over_slots_to_new_session: true\n"
+        "actions:\n"
+        "- respond_chitchat\n"
+        "- utter_greet\n"
+        "- utter_cheer_up"
+    )
+
+    expected_domain = (
+        "session_config:\n"
+        "  session_expiration_time: 60\n"
+        "  carry_over_slots_to_new_session: true\n"
+        "actions:\n"
+        "- utter_chitchat\n"
+        "- utter_greet\n"
+        "- utter_cheer_up\n"
+        "version: '2.0'\n"
```

Comment:
Why is the `version` key on the last line? 🤔

Reply:
because it's added last to the dictionary before it's returned... Happy to change it in another PR though, I fear it could break other tests. WDYT?

Reply:
done in ef7686a021a, let's see if some tests fail ⏳

Reply:
should I add a changelog fragment for this?